package.json

```
@@ -14,6 +14,7 @@
   "scripts": {
     "build": "rm -rf dist/* && microbundle -o dist/index.js -f modern,umd --external react=React --sourcemap false --jsx React.createElement --no-compress --tsconfig ./tsconfig.json",
     "dev": "rm -rf dist/* && microbundle watch -o dist/index.js -f modern,umd --external react=React --sourcemap false --jsx React.createElement --no-compress --tsconfig ./tsconfig.json",
+    "start": "npm run barrel:watch && npm run dev",
```

I though about this again. I think we should run the barrel task also before the build process.
```suggestion
    "build": "npm run barrel && npm run microbundle:build",
    "start": "npm run barrel:watch && npm run mircobundle:dev",
    "microbundle:build": "rm -rf dist/* && microbundle -o dist/index.js -f modern,umd --external react=React --sourcemap false --jsx React.createElement --no-compress --tsconfig ./tsconfig.json",
    "microbundle:dev": "rm -rf dist/* && microbundle watch -o dist/index.js -f modern,umd --external react=React --sourcemap false --jsx React.createElement --no-compress --tsconfig ./tsconfig.json",
```